service/src/main/java/bio/terra/cbas/controllers/RunsApiController.java

```diff
@@ -1,20 +1,69 @@
 package bio.terra.cbas.controllers;
 
 import bio.terra.cbas.api.RunsApi;
+import bio.terra.cbas.config.CromwellServerConfiguration;
 import bio.terra.cbas.model.RunState;
 import bio.terra.cbas.model.RunStateResponse;
+import cromwell.client.ApiClient;
+import cromwell.client.api.WorkflowsApi;
 import java.util.UUID;
+import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.http.HttpStatus;
 import org.springframework.http.ResponseEntity;
 import org.springframework.stereotype.Controller;
 
 @Controller
 public class RunsApiController implements RunsApi {
 
+  private final CromwellServerConfiguration cromwellConfig;
+
+  @Autowired
+  public RunsApiController(CromwellServerConfiguration cromwellConfig) {
+    this.cromwellConfig = cromwellConfig;
+  }
+
   @Override
   public ResponseEntity<RunStateResponse> postRun(String workflowUrl, Object workflowParams) {
+
+    ApiClient client = new ApiClient();
+    client.setBasePath(this.cromwellConfig.baseUri());
+    client.setDebugging(true);
+    client.setLenientOnJson(true);
```

Comment:
What does `setLenientOnJson` do?

Reply:
When debugging this allowed me to actually see not-JSON error output, instead of just getting a "this is not JSON!" error message. Removed.